web/docs/deployment/ci-cd.md

```diff
@@ -1,7 +1,113 @@
 ---
 title: CI/CD Scenarios
 ---
+Setting up a CI/CD pipeline is an optional but highly recommended part of deploying applications.
 
-TODO: we'll talk about how to e2e test Wasp apps in the CI
+**Continuous Integration (CI)** involves verifying/testing code changes through an automated process whenever code is pushed to the repository. This helps us catch bugs early and make sure that our app works.
 
-TODO: we'll talk about building the server as a Docker image in the CI
\ No newline at end of file
+**Continuous Deployment (CD)** refers to the automatic deployment of code changes to the production environment. This is commonly know as "push to deploy" and frees developers from having to manually deploy code changes.
+
+## Running tests in CI
```

Comment:
I wonder if this should be under deployment, since CI is a separate thing from deployment.

Maybe it would be better if we had a part of docs called Testing and then tehre we can have CI part?